eng/common/SetupNugetSources.ps1

```diff
@@ -35,7 +35,7 @@ Set-StrictMode -Version 2.0
 . $PSScriptRoot\tools.ps1
 
 # Add source entry to PackageSources
-function AddPackageSource($sources, $SourceName, $SourceEndPoint, $creds, $Username, $Password) {
+function AddPackageSource($sources, $SourceName, $SourceEndPoint, $creds, $Username, $pwd) {
```

Comment:
Were/Are these changes supposed to come from arcade at some point?

Reply:
They were already checked in [https://github.com/dotnet/arcade/pull/14553](here). I just copy/pasted the file from the Arcade repo to clear the SDL failures in the run.